exercises/complex-numbers/ComplexNumbersTest.cs

```diff
@@ -0,0 +1,224 @@
+// This file was auto-generated based on version 1.0.0 of the canonical data.
+
+using Xunit;
+
+public class ComplexNumbersTest
+{
+    [Fact]
+    public void Imaginary_unit()
+    {
+        var z1 = new ComplexNumber { Real = 0, Imaginary = 1 };
+        var z2 = new ComplexNumber { Real = 0, Imaginary = 1 };
+        var expected = new ComplexNumber { Real = -1, Imaginary = 0 };
+        Assert.Equal(expected, ComplexNumbers.Mul(z1, z2));
```

Comment:
I was wondering of the static class approach makes sense. Shouldn't we be using instance methods?